msp/mspimplsetup.go

```diff
@@ -493,6 +493,7 @@ func (msp *bccspmsp) setupTLSCAs(conf *m.FabricMSPConfig) error {
 			return errors.WithMessagef(err, "CA Certificate problem with Subject Key Identifier extension, (SN: %x)", cert.SerialNumber)
 		}
 
+		opts.CurrentTime = cert.NotBefore.Add(time.Second)
```

Comment:
what about line 155? Can you also add a similar option there?

Reply:
Sure let me give it a look

Reply:
The thing with line 155 is that there's no validation of the expiration date after defining the msp.opts there, the validation is on the finalizeSetupCAs method, on line 260, and when I tested finalizeSetupCAs with an expired certificate, using unit tests, they passed. So I'm not sure the current time override is needed for the CA, as the unit test for finalizeSetupCAs didn't failed, or if there's an extra validation missing for the CA, where that validation should be happening.

Reply:
Oh you're right. I didn't notice we're not verifying there at all. 

It doesn't fail your second test because `finalizeSetupCAs`  passes through `getValidityOptsForCert` which has the time override.